tests/foreman/sys/test_dynflow.py

```diff
@@ -0,0 +1,41 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Dynflow
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Dynflow
+
+:Requirement: Dynflow
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo import ssh
+from robottelo.decorators import tier2
+from robottelo.test import TestCase
+
+
+class DynflowTestCase(TestCase):
+    """Dynflow tests"""
+
+    @tier2
+    def test_positive_setup_dynflow(self):
+        """Set dynflow parameters, restart it and check it adheres to them
+
+        :id: a5aaab5e-bc18-453e-a284-64aef752ec88
+
+        :expectedresults: Correct dynflow processes are running, respecting settings
+
+        :CaseImportance: High
+        """
+        ssh.command(
+            "cd /etc/foreman/dynflow/ && cp worker-hosts-queue.yml test.yml && sed -i s/5/6/"
+            "test.yml && systemctl restart 'dynflow-sidekiq@test' && res=1 && while"
+            "[ $res -eq 1 ]; do systemctl status 'dynflow-sidekiq@test' -l | "
+            "grep ' of 6 busy' && res=$(echo $?); done"
+        )  # if thread count is not respected or the process is not running, this should timeout
```

Comment:
No Python Assertion? dont you think good to check the return code? you are managing everything in the shell itself.

Reply:
I think there is nothing to check - the correct thread count either is there and the script doesn't time out or it is not there after certain waiting (depends on how long ssh.command is set up to wait) and the script times out.